server/middlewares/stats.js

```diff
@@ -24,13 +24,20 @@ module.exports = (req, res, next) => {
   const ip = req.headers['x-forwarded-for']
 
   const mydevice = device(req.headers['user-agent'])
+  const date = new Date()
 
   if (!knownIps.has(ip) && !mydevice.is('bot')) {
-    const date = new Date()
     knownIps.add(ip)
     client.incr('total')
     client.incr(`${monthNames[date.getMonth()]}-${date.getFullYear()}`)
     client.incr(`${date.getDate()}.${date.getMonth() + 1}.${date.getFullYear()}`)
   }
+
+  if (req.originalUrl === '/api/txs/submit' && req.method === 'POST') {
+    client.incr('total tx submissions')
+    client.incr(`${monthNames[date.getMonth()]}-${date.getFullYear()} (txs submitted)`)
+    client.incr(`${date.getDate()}.${date.getMonth() + 1}.${date.getFullYear()} (txs submitted)`)
```

Comment:
I think it would be better to use some kind of code words so it will be more easy to parse later